src/main/java/lotto/domain/LottoStore.java

```diff
@@ -1,24 +1,36 @@
 package lotto.domain;
 
+import lotto.exception.ErrorMessage;
+
 import java.util.ArrayList;
 import java.util.List;
 import java.util.stream.Collectors;
 
 //구매 금액만큼 로또를 발급하는 역할을 가진 클래스
 public class LottoStore {
 
+    public static final int LOTTO_PRICE = 1000;
+
     private LottoStore() {
     }
 
-    public static Lottos purchase(Money money) {
-        List<Lotto> lottos = createLottos(money.divide());
+    public static Lottos purchase(final Money money, final int manualPurchaseCount, final List<List<Integer>> manualLottoNumber) {
```

Comment:
manualLottoNumber.size()가 수동 로또의 개수를 대체할 수도 있을 것 같습니다 :)
```suggestion
    public static Lottos purchase(final Money money,  final List<List<Integer>> manualLottoNumber) {
```

Reply:
manualLottoCountCheck() 메서드에서 사용자가 입력한 구매개수와 입력한 수동 로또에 대한 개수를 체크를 하고 있어서 manualPurchaseCount로 받아야할 것 같습니다 :)